translation/dest/site/ca-ES.xml

```diff
@@ -506,7 +506,7 @@
   <string name="firstName">Nom</string>
   <string name="lastName">Cognoms</string>
   <string name="biography">Biografia</string>
-  <string name="flag" comment="flag&#10;Appears when you edit your profile. Lets a player add a country/flag (not all flags are for countries) to their profile">Bandera</string>
+  <string name="countryRegion" comment="flag&#10;Appears when you edit your profile. Lets a player add a country/flag (not all flags are for countries) to their profile">País o regió</string>
```

Comment:
```suggestion
  <string name="countryRegion" comment="flag&#10;Appears when you edit your profile. Lets a player add a country/flag (not all flags are for countries) to their profile">País o región</string>
```

Reply:
To contribute to the translations, click here: https://crowdin.com/project/lichess
this pull request is automatically generated from there

Reply:
Thanks